### DISCUSSION TITLE
[SUGGESTION] Stations not secure from clipping hackers

### DISCUSSION THREAD

**Discussion by cloudrambler** [tsunamayo/Starship-EVO#4465] [Suggestions]
The last time I explored the tutorial station (maybe two weeks ago) I used F5 to clip through and see what was hidden inside the structures. Once there it was possible to interface with the gravity generator and turn it off for the whole station. That felt like it was an oversight as the rest of the bricks were not interactive to build or delete.

So I think the faction restrictions maybe don't cover interfacing certain control devices, which perhaps they should. Ideally only switches and kiosks would allow multi-faction interfacing, with all computers, logic, and mechanism (direct action) being isolated to build faction.

**Comment by tsunamayo** [comment]
@cloudrambler Hi, yes I plan to add some locking mechanism to each ui, that could allow only some player or some faction to access the panels. Thanks
